Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -108,9 +106,10 @@ struct MessagesView_Previews: PreviewProvider {
             [
                 Chat(role: .system, content: "System Message!"),
                 Chat(role: .system, content: "System Message (hidden)!"),
+                Chat(role: .function, content: "Function Message!"),
                 Chat(role: .user, content: "User Message!"),
                 Chat(role: .assistant, content: "Assistant Message!")
-            ]
+            ], hideMessagesWithRoles: MessageView.Defaults.hideMessagesWithRoles
```

Comment:
If you use the default parameters above, you could omit the argument here:
```suggestion
            ]
```